engines/ultima/ultima8/graphics/palette_manager.cpp

```diff
@@ -155,6 +155,15 @@ void PaletteManager::untransformPalette(PalIndex index) {
 	transformPalette(index, matrix);
 }
 
+bool PaletteManager::getTransformMatrix(int16 matrix[12], PalIndex index) {
+	Palette *pal = getPalette(index);
+
+	if (!pal) return false;
+
+	for (int i = 0; i < 12; i++) matrix[i] = pal->_matrix[i];
```

Comment:
Please follow our coding guidelines, and don't put these in one line

Reply:
Sure. I don't like this style either and was just following the other loops in the file. Since there's only 2 others I'll just fix them all.

Reply:
Thanks. Properly seperating lines like that from the original Pentagram source is an outstanding cleanup. I'm hopeful that it can done in bulk, but at least running astyle using the provided parameters on the Wiki didn't do it. And I'm wary of accidentally introducing errors if I tried to do it manually file by file.